news/6641.fix

```diff
@@ -0,0 +1 @@
+Changed looks_like_hash to use pytest paraters
```

Comment:
This is a trivial fix so shouldn't have a news entry: https://pip.pypa.io/en/stable/development/contributing/#choosing-the-type-of-news-entry